cdap-docs/user-guide/source/pipelines/plugins/actions/index.rst

```diff
@@ -18,5 +18,5 @@ These plugins work with:
     :maxdepth: 1
     :glob:
 
-    **
+    *
```

Comment:
This is pattern taken from JJ's work which auto-populates TOC with files in dir.